pset-functions.js

```diff
@@ -115,6 +155,14 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+
+
+const isPalindrome = word => word === word.split('').reverse().join('');
+
```

Comment:
This was such a well written and concise code I almost missed it! Really great use of method chaining here.